### ISSUE TITLE
Request for truncated normal distribution

### ISSUE THREAD

**Issue by taless474** [STEllAR-GROUP/phylanx#864]
A [truncated normal](https://keras.io/backend/#truncated_normal) has values generated follow a normal distribution with specified mean and standard deviation, except that values whose magnitude is more than two standard deviations from the mean are dropped and re-picked.
It should be able to accept `shape`, `mean`, `stddev`, `dtype` and `seed`
